Car.cpp

```diff
@@ -3,15 +3,16 @@
 #include <iostream>
 
 Car::Car(const std::string& n) : Vehicle(n) {}
+// Car::Car(Car const&) : Vehicle() {}
 
 void Car::closeWindows()
 {
-    setSpeed(70);
+    setSpeed(100);
```

Comment:
Don't change code that doesn't need to be changed.
be surgical with your edits and revisions when you work on other people's code.
- [ ] revert